packages/webviz-core/src/panels/ImageView/ImageCanvas.js

```diff
@@ -89,6 +89,9 @@ export default class ImageCanvas extends React.Component<Props, State> {
           case "bgr8": decodeBGR(rawData, width, height, image.data); break;
           case "32FC1": decodeFloat1c(rawData, width, height, image.data); break;
           case "bayer_rggb8": decodeRGGB(rawData, width, height, image.data); break;
+          case "mono8": decodeMono8(rawData, width, height, image.data); break;
```

Comment:
nit: can make it clear these two are treated the same by using a fallthrough:
```suggestion
          case "mono8":
```